.github/workflows/refresh-examples.yml

```diff
@@ -0,0 +1,31 @@
+name: refresh-examples
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.sha }}'
+  cancel-in-progress: true
+
+on:
+  pull_request:
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    name: Refresh examples
+    env:
+      gouttelette: "./gouttelette"
```

Comment:
We shouldn't use gouttelette anymore! Am I wrong @GomathiselviS? @abikouo

Reply:
yes gouttelette is removed but for now I am just migrating the job as it was on zuul, the we need to update the `tox.ini` file along with the github action with the new tool

Reply:
This workflow is not needed anymore, as these tests will be run by Ansible Content Builder tool.